Files/MultilingualResources/Files.hu-HU.xlf

```diff
@@ -1576,11 +1576,11 @@
         </trans-unit>
         <trans-unit id="PropertyPromotionUrl" translate="yes" xml:space="preserve">
           <source>Promotion Url</source>
-          <target state="new">Promotion Url</target>
+          <target state="translated">Promotion Url</target>
```

Comment:
Only translated strings should be marked as translated. Keep untranslated strings with state 'new'.

Reply:
Ok, I'm starting to understand.